storybook/stories/spacingNames/default.js

```diff
@@ -0,0 +1,19 @@
+// @flow
```

Comment:
This falls within the other usage examples I would put it in there rather than on a new section

Reply:
I think having a separate story with notes would be really helpful.  I'll put it under the Grid folder.  This is related to both the margin and spacing story but I don't want to put it in both or just one of them.